spec/output/generator_spec_elixir/nesting.x/my_union.ex

```diff
@@ -0,0 +1,74 @@
+defmodule MyXDR.MyUnion do
+  @moduledoc """
+  Automatically generated by xdrgen
+  DO NOT EDIT or your changes may be overwritten
+
+  Target implementation: elixir_xdr at https://hex.pm/packages/elixir_xdr
+
+  Representation of Stellar `MyUnion` type.
+  """
+
+  @behaviour XDR.Declaration
+
+  alias MyXDR.{UnionKey,
+    MyUnionOne,
+    MyUnionTwo,
+    XDR.Type.Void
+  }
+
+  @arms [
+    ONE: MyUnionOne,
+    TWO: MyUnionTwo,
+    OFFER: XDR.Type.Void
+  ]
+
+  @type value ::
+    MyUnionOne.t()
+    | MyUnionTwo.t()
```

Comment:
Let's add proper indenting
```suggestion
  @type value ::
          MyUnionOne.t()
          | MyUnionTwo.t()
```